orchestrator/build.gradle

```diff
@@ -30,33 +30,37 @@ repositories {
 }
 
 dependencies {
-    compile project(":profile")
-    compile project(":core")
-    compile project(":output")
-    compile project(":common")
-    compile project(":custom")
-
-    compile group: "info.picocli", name: "picocli", version: "${PICOCLI_VERSION}"
-    compile group: "com.google.code.gson", name: "gson", version: "${GSON_VERSION}"
-
-    testCompile project(":common").sourceSets.test.output
-    testCompile "org.hamcrest:java-hamcrest:${HAMCREST_VERSION}"
-    testCompile "org.junit.jupiter:junit-jupiter-api:${JUNIT_JUPITER_VERSION}"
-    testCompile "org.junit.platform:junit-platform-runner:${JUNIT_PLATFORM_RUNNER_VERSION}"
-    testCompile "org.junit.vintage:junit-vintage-engine:${JUNIT_JUPITER_VERSION}"
-    testCompile "org.junit.jupiter:junit-jupiter-params:${JUNIT_JUPITER_VERSION}"
-    testCompile "io.cucumber:cucumber-junit:${CUCUMBER_VERSION}"
-    testCompile "io.cucumber:cucumber-java:${CUCUMBER_VERSION}"
-    testCompile "io.cucumber:cucumber-java8:${CUCUMBER_VERSION}"
-    testCompile "io.cucumber:gherkin:${GHERKIN_VERSION}"
-    testCompile "io.cucumber:cucumber-picocontainer:${CUCUMBER_PICOCONTAINER_VERSION}"
-    testCompile "org.mockito:mockito-all:${MOCKITO_VERSION}"
-    testCompile "com.shazam:shazamcrest:${SHAZAMCREST_VERSION}"
-    testCompile "org.junit.jupiter:junit-jupiter-engine:${JUNIT_JUPITER_VERSION}"
-    testCompile group: "junit", name: "junit", version: "${JUNIT_4_VERSION}"
-
-    testCompile project(':core').sourceSets.test.output
-    testCompile project(':profile').sourceSets.test.output
+    implementation project(":profile")
+    implementation project(":core")
+    implementation project(":output")
+    implementation project(":common")
+    implementation project(":custom")
+
+    implementation group: "info.picocli", name: "picocli", version: "${PICOCLI_VERSION}"
+    implementation group: "com.google.code.gson", name: "gson", version: "${GSON_VERSION}"
+
+    implementation group: "com.google.inject", name: "guice", version: "${GUICE_VERSION}"
+    implementation "com.fasterxml.jackson.core:jackson-core:${JACKSON_VERSION}"
+    implementation "com.fasterxml.jackson.core:jackson-databind:${JACKSON_VERSION}"
+
```

Comment:
It looks like previously these jackson dependencies were getting pulled in via the core build.gradle, but now they're not.

I don't know why this has changed.